### PR TITLE
Add optional chain for iframe in _removeTransact

### PR DESCRIPTION
**Pull request by Tr3yb0**
[Linear Ticket](https://linear.app/atomicbuilt/issue/CO-63/-removetransact-erroring-if-container-removed-after-onfinish)

### CODE REVIEW

**Review by wilsonparson — APPROVED** [review]
Strange that prettier made a bunch of formatting changes

File: index.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ let atomicSDK = {
         inSdk: true,
         platform: {
           name: 'browser',
-          sdkVersion: '3.0.0',
+          sdkVersion: '3.0.2',
```
Comment (wilsonparson) [inline index.js:49]: @sean-hill When and why does this need to be updated?

Comment (sean-hill) [reply index.js:49]: @wilsonparson it's an NPM package, so whenever you make a change and want to publish, we need to bump the package version, and then we run `npm publish --access public`. This looks great!

Comment (Tr3yb0) [reply index.js:49]: @sean-hill I dont think this particular value is driven off the publish command. It was out of date with what was on npm. Problem for another day though.

Comment (sean-hill) [reply index.js:49]: @Tr3yb0 you're right. This line and the line in `package.json` should match 😄 Also, I still think what the customer is doing is dumb 😏

**Pull request merged**